ccpp/data/GFS_typedefs.meta

```diff
@@ -6354,8 +6360,8 @@
   dimensions = ()
   type = logical
 [do_gsl_drag_ls_bl]
-  standard_name = flag_for_gsl_drag_suite_large_scale_orographic_and_blocking_drag
-  long_name = flag to activate GSL drag suite - large-scale GWD and blocking
+  standard_name = flag_for_gsl_drag_suite_mesoscale_orographic_and_blocking_drag
```

Comment:
Similarly, could you change this to "do_gsl_drag_suite_mesoscale_orographic_and_blocking_drag". We're trying to switch the names of these kind of flags to be more descriptive to match other CCPP models.

Reply:
Updated code and pushed to PR.